tests/testhelpers/StatsVerify.go

```diff
@@ -0,0 +1,97 @@
+package testhelpers
+
+import (
+	"bytes"
+	"strings"
+	"reflect"
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/scootdev/scoot/common/stats"
+)
+
+/*
+add new Checker functions here as needed
+ */
+/*
+errors if a is not float64, returns true if a == b
+ */
+func FloatEqTest(a, b interface{}) bool {
+	if b == nil && a == nil {
+		return true
+	}
+	aflt := a.(float64)
+	bflt := b.(float64)
+	return aflt == bflt
+}
+/*
+errors if a is not float64, returns true if a > b
+ */
+func FloatGTTest(a, b interface{}) bool {
+	if b == nil && a == nil {
+		return true
+	}
+	aflt := a.(float64)
+	bflt := b.(float64)
+	return aflt > bflt
+}
+/*
+errors if a is not int64, returns true if a == b
+ */
+func Int64EqTest(a, b interface{}) bool {
+	if b == nil && a == nil {
+		return true
+	}
+	aint := a.(int64)
+	bint := b.(int)
+	return aint == int64(bint)
+}
+func DoesNotExist(a, b interface{}) bool {
+	return a == nil
+}
+
+/*
+defines the condition checker to use to validate the measurement.  Each Checker(a, b) implementation
+will expect a to be the 'got' value and b to be the 'expected' value.
+ */
+type Rule struct {
+	Checker func(interface{}, interface{}) bool
+	Value interface{}
+}
+
+/*
+Verify that the stats registry object contains values for the keys in the contains map parameter and that
+each entry conforms to the rule (condition) associated with that key.
+ */
+func VerifyStats(statsRegistry stats.StatsRegistry, t *testing.T, contains map[string]Rule) {
```

Comment:
if this was moved to common/stats/stats_test.go, I *think* we could keep finagleStatsRegistry private.